pages/cloud/using.md

```diff
@@ -2,13 +2,14 @@
 
 ### Managing your Project with the Cloud Dashboard
 
-Allocations to the LakeEffect Cloud are requested and managed through the ColdFront Resource Allocations Management Tool. 
+Allocations to the LakeEffect Cloud are requested and managed through the ColdFront Resource Allocations Management Tool.
 To request an allocation to LakeEffect [see here](../getting-access.md#allocation-requests).  
 Once approved through ColdFront you will be informed when your cloud access is ready and you will be able to login in to the LakeEffect Web Console
 where you can manage your project's cloud resources and view information on your group's usage.  
 
```

Comment:
The change I intended to make was to use the same VPN warning message across all our docs.  It looks like the editing tool I use also removed all the extra spaces at end of lines.